pallets/foreign-investments/Cargo.toml

```diff
@@ -30,9 +30,12 @@ sp-std = { git = "https://github.com/paritytech/substrate", default-features = f
 frame-benchmarking = { git = "https://github.com/paritytech/substrate", default-features = false, optional = true, branch = "polkadot-v0.9.38" }
 
 [dev-dependencies]
+rand = "0.8"
```

Comment:
No `*/std` needed for either of these 2 pallets?

Reply:
Good point! I think so

Reply:
I usually do not think about `std` feature if the crate doesn't come from substrate 😅

Reply:
As they are `dev-dependencies`, we don't need to disable default features. If we did, we would have to configure `std` for these of course.